### ISSUE TITLE
Show reply rate in profile

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#421]
"Replied to % of messages."

Same rules apply as to _"Show average time to reply in profile"_ #420

---

It could look like this:

```html
<div class="profile-sidebar-section text-muted">
  Reply rate 90%.<br>
  Replies within a day.
</div>
```

![image](https://cloud.githubusercontent.com/assets/87168/20387253/e2ca2b5c-acc7-11e6-875c-a086e0d56b6d.png)

**Comment by mrkvon** [comment]
implemented with #436 and #448

**Comment by mrkvon** [comment]
implemented with #436 and #448
